samples/wifi/scan/Kconfig

```diff
@@ -16,21 +16,4 @@ config WIFI_MAC_ADDRESS
 		use the default MAC address (locally administrative). This is only used
 		when the nRF7002 OTP is not programmed with a MAC address.
 
-choice  WIFI_SCAN_TYPE_SELECT
-	prompt "Scan type"
-	default WIFI_SCAN_TYPE_ACTIVE
-	help
-		Wi-Fi scan type to be used to perform current scan operation.
-		If this is not set default action is to do active scan.
-
-config WIFI_SCAN_TYPE_ACTIVE
-	bool "Active scan"
-	help
-	  Enable active scan
-
-config WIFI_SCAN_TYPE_PASSIVE
-	bool "Passive scan"
-	help
-	  Enable passive scan
-endchoice
```

Comment:
Why is this removed? The commit log itself is unclear.

Reply:
No idea, how it got removed. Added the changes.